.github/scripts/parse_rpt.py

```diff
@@ -1,33 +1,62 @@
 import sys
+import json
+import os
+import re, subprocess
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+from common.check_gen_files import check_gen_files
 
 sys.stdout.flush()
 
-if len(sys.argv) == 1 or sys.argv[1] == "sky130hvl_ldo":
+dir_path = r'flow/reports'
+lib = os.listdir(dir_path)
+
+if (len(sys.argv) > 1) and (sys.argv[1] == str(lib[0])):
+    drc_filename = "flow/reports/" + sys.argv[1] + "/cryo/6_final_drc.rpt"
```

Comment:
@chetanyagoyal : I am not sure about this tbh. The `if` block and the last `else` does the same thing. I don't know why you are considering the drc report from the `flow/reports/...` path since the reports will be copied to the `work` folder anyways in the end. If you consider that, they you don't need the first `if` and the last `else` should take care of the cryo-gen reports check